cmd/kubeadm/app/componentconfigs/kubelet.go

```diff
@@ -217,6 +217,10 @@ func (kc *kubeletConfig) Default(cfg *kubeadmapi.ClusterConfiguration, _ *kubead
 			kc.config.CgroupDriver = driver
 		}
 	}
```

Comment:
#99647 need to be updated.

Need I cherry-pick it to this PR and make the test pass?

Reply:
i can update https://github.com/kubernetes/kubernetes/pull/99647
but we can leave it for later in the cycle.

it has a few implications and needs "ACTION REQUIRED":
- it may break our kubeadm/kinder e2e test (i need to check).
- it will probably break uses that are not setting their explicit cgroup group driver.

actually let's add ACTION REQUIRED prefix to this PR's release note as well:
> ACTION REQUIRED: kubeadm: during "kubeadm init/join/upgrade", always default the "cgroupDriver" value in the KubeletConfiguration to "systemd", unless the user was explicit about the value. See https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/ for more details.

Reply:
Got it.